IBAnimatable/TransitionAnimationType.swift

```diff
@@ -202,6 +228,24 @@ extension TransitionAnimationType {
     case none
     case open
     case close
+
+    var opposite: HollowState {
+      switch self {
+      case .close: return .open
+      case .open: return .close
+      case .none:
+        fatalError("No opposing HollowState for HollowState.none")
+      }
+    }
+
+    var opposingGesture: InteractiveGestureType.GestureDirection {
+      switch self {
+      case .open: return .close
+      case .close: return .open
+      case .none:
+        fatalError("No opposing GestureDirection for HollowState.none")
```

Comment:
EDIT: 
The HollowState `opposite` property is the only type this would work for

There is no `opposingGesture` in GestureDirection for the `.none` case

Also not all `TransitionAnimationType.Direction` cases have a matching or opposing InteractiveGestureType.GestureDirection

Basically, there's not a 1 : 1 mapping between these two types. 
So we use `fatalError` if someone is trying to use something that has no counterpart.